### ISSUE TITLE
Can not add a credential

### ISSUE THREAD

**Issue by wowaTYPO3** [koninklijke-collective/koning_instagram#3]
I followed every step in the installation guide, but if I try to add a credential, I got the following error:

**An error has occured contacting the Instagram API.**

The client ID, client secret etc. are correct. Do you have any hints where to find the problem?

**Comment by jp1987** [comment]
I just uploaded version 1.0.1 to the TER that should show the actual error message whenever this goes wrong. Let me know what it says.

**Comment by wowaTYPO3** [comment]
Ok, I cloned the repo now ;)

Now the error message is:
The error message is: GuzzleHttp Client not included, please run composer with "guzzlehttp/guzzle": requirement. "composer require guzzlehttp/guzzle"

I installed guzzle before, it is in the directory  vendor/guzzlehttp/guzzle/ in my project-directory.

**Comment by jp1987** [comment]
You need to make sure vendor/autoload.php is included into your project, otherwise Composer dependencies cannot be loaded. I’d recommend including it from typo3conf/AdditionalConfiguration.php, like this:

```
<?php
// Load the autoload for composer
if (file_exists(PATH_site . 'vendor/autoload.php')) {
   require_once(PATH_site . 'vendor/autoload.php');
}
```

**Comment by wowaTYPO3** [comment]
Ok, now it works! Thank you! Could you add these lines of code in the documentation? Not everyone is familiar with composer ;)

It is mandatory to specify one ore more tags in the plugin. Would it be possible to do it without tags and simply show all images of the instagram account?

**Comment by jp1987** [comment]
Ok, I will.

That's currently not possible, but I'll look into building a separate plugin for showing all contents of the linked credential. However, this is a spare time project, so it will be sometime on a weekend ;)

**Comment by jp1987** [comment]
Just released 1.1.0, should show up in TER soon. This version has a plugin to list media by a credential (so for a user). Also added the documentation for composer. Thanks for the feedback.

**Comment by jp1987** [comment]
Just to let you know. the composer dependency is now included in the extension itself. No need to load it separately anymore.
